### ISSUE TITLE
SIGNATURE VERIFICATION FAILED!

### ISSUE THREAD

**Issue by spectra122** [micahflee/torbrowser-launcher#562]
![Снимок экрана от 2021-06-12 12-35-41](https://user-images.githubusercontent.com/62400327/121771899-c1e1d080-cb61-11eb-985e-936c281d1149.png)

**Comment by spectra122** [comment]
But when I changed the localization of the interface back into English, the problem disappeared .

**Comment by greglarsen** [comment]
I got the same error.  uname -a:
 Linux opus 5.11.0-18-generic #19-Ubuntu SMP Fri May 7 14:22:03 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

My installation is English.

**Comment by mhnd-lab** [comment]
I have the same issue on Debian buster installed via the backports

**Comment by jisom38903** [comment]
#metoo

**Comment by jignac05** [comment]
I have the same error message

**Comment by mingcenwei** [comment]
The cause of this issue actually lies in Tor project instead of this project. The GPG signing subkey `rsa4096/EB774491D9FF06E2` expired on `2021-06-12`:
```shell
$ gpg --list-options show-unusable-subkeys --list-public-keys
...
pub   rsa4096/4E2C6E8793298290 2014-12-15 [C] [expires: 2025-07-21]
      Key fingerprint = EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
uid                 [ unknown] Tor Browser Developers (signing key) <torbrowser@torproject.org>
sub   rsa4096/2D000988589839A3 2014-12-15 [S] [revoked: 2015-08-26]
      Key fingerprint = 05FA 4425 3F6C 19A8 B7F5  18D4 2D00 0988 5898 39A3
sub   rsa4096/EB774491D9FF06E2 2018-05-26 [S] [expired: 2021-06-12]
      Key fingerprint = 1107 75B5 D101 FB36 BC6C  911B EB77 4491 D9FF 06E2

$ gpg --verify ~/.cache/torbrowser/download/tor-browser-linux64-10.0.17_en-US.tar.xz.asc.verification_failed ~/.cache/torbrowser/download/tor-browser-linux64-10.0.17_en-US.tar.xz.verification_failed
gpg: Signature made Wed 02 Jun 2021 07:26:05 AM CST
gpg:                using RSA key EB774491D9FF06E2
gpg: Good signature from "Tor Browser Developers (signing key) <torbrowser@torproject.org>" [unknown]
gpg: torbrowser@torproject.org: Verified 1 signatures in the past 21 hours.
     Encrypted 0 messages.
gpg: Note: This key has expired!
Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
     Subkey fingerprint: 1107 75B5 D101 FB36 BC6C  911B EB77 4491 D9FF 06E2
```

**Comment by jisom38903** [comment]
yea i know, it downloads the same tarball and sig on torproject. i just want to complain here

**Comment by mingcenwei** [comment]
See #564 for a workaround.

**Comment by marjia123** [comment]
what's the solution?

**Comment by jisom38903** [comment]
> what's the solution?

adjust your clock or just download it from the torproject site

**Comment by mathusaelm** [comment]
can we just wait for an update key in a few ?

**Comment by jisom38903** [comment]
> can we just wait for an update key in a few ?

we can. but can you?

**Comment by micahflee** [comment]
Fixed in torbrowser-launcher 0.3.4
